main.go

```diff
@@ -0,0 +1,88 @@
+package main
+
+import (
+	"bufio"
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/appc/docker2aci/lib"
+	"github.com/coreos/rocket/cas"
+)
+
+const (
+	rocketDir = "/var/lib/rkt"
+)
+
+var flagImport = flag.Bool("import", false, "Import ACI images to the rocket store")
```

Comment:
@jonboulle I'm assuming coreos/rocket#145 will just re-use this library? I certainly feel a standalone docker2aci cmd is useful as part of the appc project (separate from rocket)